src/lib/storage/vector_compression/simd_bp128/simd_bp128_iterator.hpp

```diff
@@ -23,6 +23,13 @@ class SimdBp128Iterator : public BaseCompressedVectorIterator<SimdBp128Iterator>
   SimdBp128Iterator(SimdBp128Iterator&& other) = default;
   ~SimdBp128Iterator() = default;
 
+  SimdBp128Iterator& operator=(const SimdBp128Iterator& other) {
+    // TODO(cmfcmf): The code won't compile without this method.
+    // I am a) unsure why this method is needed and b) it has to be implemented correctly.
+    Fail("Not implemented");
+    return *this;
+  }
```

Comment:
How about `SimdBp128Iterator& operator=(const SimdBp128Iterator&) = default;` if you cannot find a way to avoid it's implementation?